src/en/infinityscans/src/eu/kanade/tachiyomi/extension/en/infinityscans/InfinityScans.kt

```diff
@@ -49,59 +47,106 @@ class InfinityScans : HttpSource() {
         add("Sec-Fetch-Dest", "empty")
         add("Sec-Fetch-Mode", "cors")
         add("Sec-Fetch-Site", "same-origin")
-        add("X-Requested-With", "XMLHttpRequest")
+        add("X-requested-with", "XMLHttpRequest")
```

Comment:
Why change? Casing doesn't matter for HTTP/2 in any case

Reply:
No thoughts. I didn't know http specification, and I was seeing that it sends small. So I changed it.